src/overlay005/fieldmap.c

```diff
@@ -426,10 +426,10 @@ static BOOL FieldMap_ChangeZone(FieldSystem *fieldSystem)
     }
 
     {
-        int v5 = MapHeaderData_GetNumObjectEvents(fieldSystem);
-        const ObjectEvent *v6 = MapHeaderData_GetObjectEvents(fieldSystem);
+        int npcCount = MapHeaderData_GetNumObjectEvents(fieldSystem);
```

Comment:
**suggestion:** I see we do use NPC in a few places in the repo right now, but it feels a bit inaccurate to me. I feel like it doesn't encapsulate the full range of object events, and it's not uniform with the struct field it retrieves. I think I'd go with `objEventCount` or something, for example.